prime-router/src/main/kotlin/azure/SendFunction.kt

```diff
@@ -29,14 +29,27 @@ import kotlin.random.Random
 /**
  * Azure Functions with HTTP Trigger. Write to blob.
  */
-const val dataRetentionDays = 7L
 const val send = "send"
-const val maxRetryCount = 4
-const val maxDurationValue = 120L
+// needed in case retry index is out-of-bounds call but should not be every be used
+const val defaultMaxDurationValue = 120L
+
+// exact time of next retry is slightly randomized to avoid the situation when
+// there's a system-wide failure and everything that's failed retries at the exact
+// same time producing a spike that makes things worse.
+// This is +/- around the actual retry, so they are spread out by up to 2x the value
+// It should always be > than the first entry of the retryDurationInMin above
+const val initialRetryInMin = 10
+const val ditherRetriesInSec = (initialRetryInMin / 2 * 60)
```

Comment:
"dither" was hard coded to `Random.nextInt(-30, 31)` Moved into constant value and documented.